themes/custom/lex/scss/molecules/_event_card.scss

```diff
@@ -0,0 +1,85 @@
+.event-card {
+    height: auto;
+    width: 300px;
+    background-color: color(med-blue);
+    color: color(white);
+
+    img {
+        height: 25px;
+        width: 35px;
+    }
+    .button-heading {
+        margin: 0 0 15px 0;
+        color: color(white);
+    }
+    .row {
+        padding-top: 15px;
+    }
+
+    .button-heading {
+        border-bottom: 1px solid color(white);
+    }
+
+    .description {
+        font-size: 1rem;
+    }
+
+    .button-icon {
+        font-size: 1.6em;
+    }
```

Comment:
These are fine for now but let's try using the appropriate h1-h4 tags for font-sizes (`<div><h1></div>`) instead of just using like `div.class`... it keeps us from having font-size styling all over the place. Nick did a nice job trying to think about that.